modules/tenant-management/src/Volo.Abp.TenantManagement.Domain/Volo/Abp/TenantManagement/TenantConfigurationCacheItemInvalidator.cs

```diff
@@ -37,6 +44,7 @@ protected virtual async Task ClearCacheAsync(Guid? id, string normalizedName)
             {
                 TenantConfigurationCacheItem.CalculateCacheKey(id, null),
                 TenantConfigurationCacheItem.CalculateCacheKey(null, normalizedName),
+                TenantConfigurationCacheItem.CalculateCacheKey(id, normalizedName),
```

Comment:
As far as I know, there is no such cache. It could only be an `id` or a `normalizedName`.

Reply:
If someone directly uses the [TenantConfigurationCacheItem](https://github.com/abpframework/abp/blob/5eba0b7ac1f15adf1245a102ae7eb5f3a955a734/framework/src/Volo.Abp.MultiTenancy.Abstractions/Volo/Abp/MultiTenancy/TenantConfigurationCacheItem.cs#L23) and send two parameters there might be cache like this.